sources/ClangSharp/Extensions/CXModuleMapDescriptor.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+
+namespace ClangSharp
+{
+    public unsafe partial struct CXModuleMapDescriptor : IDisposable
```

Comment:
Had to add a couple more "safe" wrappers so the non-PInvokeGenerator unit tests would pass.